Source/WebKit/UIProcess/WebPageProxyTesting.cpp

```diff
@@ -80,7 +80,7 @@ void WebPageProxyTesting::setDefersLoading(bool defersLoading)
 
 void WebPageProxyTesting::dispatchActivityStateUpdate()
 {
-    RunLoop::current().dispatch([protectedPage = Ref { m_page.get() }] {
+    RunLoop::current().dispatch([protectedPage = protectedPage()] {
```

Comment:
Doesn't seem to make a difference? I think the previous code is a more usual pattern in WebKit?